var/spack/repos/builtin/packages/libnrm/package.py

```diff
@@ -18,6 +18,7 @@ class Libnrm(AutotoolsPackage):
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool', type='build')
+    depends_on('pkg-config', type='build')
```

Comment:
```suggestion
    depends_on('pkgconfig', type='build')
```
This makes it a virtual dependency either satisfied by `pkg-config` or `pkgconf`(the default).

Reply:
roger that. changes incorporated. @adamjstewart